nationality-park/park.js

```diff
@@ -0,0 +1,50 @@
+import parksData from "./nationalParksOfPolandData.json" assert { type: "json" };
+
+const queryString = window.location.search;
```

Comment:
do you need queryString variable somewhere else than 4th line? if there is only one usage, variable is useless